server/db/models/potions.js

```diff
@@ -2,28 +2,29 @@ const Sequelize = require('sequelize')
 const db = require('../db')
 
 const Potion = db.define('potion', {
-    name: {
```

Comment:
Changes needing resolution were resolved by Harmony locally.

Reply:
This has been fixed